openfisca_uk/parameters/benefits/working_tax_credit/min_hours/couple_with_children.yaml

```diff
@@ -0,0 +1,6 @@
+description: Couple with children hours requirement
```

Comment:
```suggestion
description: Minimum weekly hours of work for couples with children to be eligible for the Working Tax Credit
```

Reply:
Is this summed between both members of the couple?

Reply:
Yep, but there's an additional requirement for at least one member to work at least 16 hours a week.